src/sentry/api/base.py

```diff
@@ -365,6 +365,10 @@ def initialize_request(self, request: HttpRequest, *args: Any, **kwargs: Any) ->
                 rv.user = orig_user
         return rv
 
+    def has_pagination(self, response: Response) -> bool:
+        # If response is paginated, it will have a "Link" header
+        return response.headers.get("Link") is not None
```

Comment:
We should verify this with the payload where there are no additional objects (to paginate). What ends up being the value and will the header still be added?

Reply:
If `self.paginate` is called, it will always add the cursor headers. 

https://github.com/getsentry/sentry/blob/master/src/sentry/api/base.py#L544


https://github.com/getsentry/sentry/blob/master/src/sentry/api/base.py#L459-L469

Reply:
Looking through the code, `self.paginate` will always call `self.add_cursor_headers`, which will always add the "Link" headers, so I think we should be good with this check.

https://github.com/getsentry/sentry/blob/master/src/sentry/api/base.py#L544
https://github.com/getsentry/sentry/blob/master/src/sentry/api/base.py#L459-L469

Reply:
We should verify if there's a test that asserts this logic, so that it doesn't silently break